packages/bodiless-components/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -236,6 +248,8 @@ const withBreadcrumbItemsFromStore = (Component: ComponentType<BreadcrumbsProps
               nodeKey: titleNodePath,
               nodeCollection,
             },
+            position: index + 1,
+            isCurrentPage: current.isLast(),
```

Comment:
Hm... is the requirement to style the *last* breadcrumb or the current page?  The last breadcrumb might be the parent of the current page, might it not?

Reply:
fixed